images/Makefile

```diff
@@ -4,7 +4,7 @@ IMAGE_JUPYTER_TAG = 0.7.1
 
 KAMU_VERSION = $(shell cargo metadata --format-version 1 | jq -r '.packages[] | select( .name == "kamu") | .version')
 
-SQLX_VERSION = 0.8.2
+SQLX_VERSION = 0.8.5
```

Comment:
Can we extract this version from `Cargo.lock` just like we do for `KAMU_VERSION`? I'd like to limit the number of hard-coded versions we have to maintain in different scripts.